it/controllers/APIControllerISpec.scala

```diff
@@ -200,170 +200,6 @@ class APIControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock wit
       |}
       |""".stripMargin)
 
-  "getSummaryDataForVRN" should {
-    "call API 1812 when call 1812 feature is enabled" must {
```

Comment:
I didn't mean to remove the whole test just the 'when the 1812 feature switch is enabled' part so it should look like:
"getSummaryDataForVRN" must {
"call API 1812 when call 1812 feature is enabled" in {
etc.